### PR TITLE
Add compression option 1 

### PR DESCRIPTION
**Pull request by kalrakhushi**
compression option 1: (key, value)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -37,6 +37,7 @@ pip-log.txt
 pip-delete-this-directory.txt
 
 # Unit test / coverage reports
+test/
```
Comment (matthiasdiener) [inline .gitignore]:
```suggestion
```

File: skvlite/skvlite.py [hunk 2]
```diff
@@ -1,103 +1,88 @@
...
                               (keyhash,))
         row = c.fetchone()
         if row is None:
             raise NoSuchEntryError(key)
 
-        stored_key, value = pickle.loads(row[0])
+        # Decompress the data
+        compressed_data = row[0]
+        pickled_data = zlib.decompress(compressed_data)
+        stored_key, value = pickle.loads(pickled_data)
```
Comment (matthiasdiener) [inline skvlite/skvlite.py]: create a new method `def _store_data(self, ....)` (same for `_load_data`)

File: test/test_skvlite.py [hunk 3]
```diff
@@ -1,343 +0,0 @@
-import shutil
```
Comment (matthiasdiener) [inline test/test_skvlite.py]: Please don't delete this file.

File: .gitignore (hunk 1 above)
Comment (kalrakhushi) [reply .gitignore]: done

File: skvlite/__init__.py [hunk 4]
```diff
@@ -228,35 +230,20 @@ def __delitem__(self, key: Any) -> None:
...
+    def __delitem__(self, key: K) -> None:
         raise AttributeError("Write-once KVStore")
+
+
+class PersistentDict(KVStore[K, V]):
+    pass
+
+
+class WriteOncePersistentDict(WriteOnceKVStore[K, V]):
+    pass
```
Comment (matthiasdiener) [inline skvlite/__init__.py]: These can be removed

File: skvlite/__init__.py [hunk 5]
```diff
@@ -2,9 +2,10 @@
 import pickle
 import sqlite3
 from typing import Any, Generator, Mapping, Optional, Tuple, TypeVar, cast
-
+import zlib
```
Comment (matthiasdiener) [inline skvlite/__init__.py:4]: Please undo the formatting changes (and perhaps run `isort`) in order to make the `flake8` test pass.

**Pull request closed without merging**